src/odb/include/odb/db.h

```diff
@@ -8269,6 +8265,8 @@ class dbModInst : public dbObject
   /// Returns new mod inst if the operations succeeds.
   /// Old mod inst is deleted along with its child insts.
   dbModInst* swapMaster(dbModule* module);
+  bool hasDbInst(dbInst* inst) const;
+  bool hasDbModInst(dbModInst* mod_inst) const;
```

Comment:
I prefer containsDbInst / containsDbModInst.  A comment that this is a recursive check would be helpful.